### PR TITLE
Fix emx documentation

### PR DESCRIPTION
**Pull request by bartcharbon**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits

### CODE REVIEW

File: docs/user_documentation/ref-emx.md [hunk 1]
```diff
@@ -158,10 +157,10 @@ Free text documentation describing the attribute
 Description for specified language (can be multiple languages, example: description-nl)
 
 ### rangeMin 
-Used to set range in case of int attributes
+Used to set range in case of int or long attributes
```
Comment (dennishendriksen) [inline docs/user_documentation/ref-emx.md]: Should also work for decimal, date and datetime

File: docs/user_documentation/ref-emx.md [hunk 2]
```diff
@@ -158,10 +157,10 @@ Free text documentation describing the attribute
...
 
 ### rangeMax 
-Used to set range in case of int attributes
+Used to set range in case of int or long attributes
```
Comment (dennishendriksen) [inline docs/user_documentation/ref-emx.md]: Should also work for decimal, date and datetime

File: docs/user_documentation/ref-emx.md [hunk 3]
```diff
@@ -255,7 +254,7 @@ free text description of the entity
 description for specified language (can be multiple languages, example: description-nl)
 
 ### backend
-the backend (database) to store the entities in (currently PostGreSQL or ElasticSearch)
+the backend (database) to store the entities in (currently only PostGreSQL)
```
Comment (dennishendriksen) [inline docs/user_documentation/ref-emx.md]: PostgreSQL (note the casing)

File: docs/user_documentation/ref-emx.md [hunk 4]
```diff
@@ -103,8 +103,7 @@ Packages:
 For all technical names in the EMX format, the following rules apply:
 - No special characters, except for; '_' and '#', only letters, numbers are allowed.
 - No names starting with digits. 
-- Maximum length for names is 30 chars.
-- Keywords used by programming languages (e.g. java, javascript, postgresql) are not allowed.
+- The keywords: "login", "logout", "csv", "base", "exist", "meta", are not allowed.
```
Comment (dennishendriksen) [inline docs/user_documentation/ref-emx.md]: Application will break for the additional keywords: version, i18n, copy

Comment (bartcharbon) [reply docs/user_documentation/ref-emx.md]: created: https://github.com/molgenis/molgenis/issues/7011

**Pull request merged**